src/structures/MessageMentions.js

```diff
@@ -70,7 +70,7 @@ class MessageMentions {
       } else {
         this.roles = new Collection();
         for (const mention of roles) {
-          const role = message.channel.guild.roles.cache.get(mention);
+          const role = message.guild.roles.cache.get(mention);
```

Comment:
We probably want to store `guild` outside of the loop, as well as guard it, so we don't execute the getter's logic multiple times.